README.md

```diff
@@ -15,3 +15,152 @@ To install Kubeflow Pipelines, run:
     juju deploy kubeflow-pipelines
 
 For more information, see https://juju.is/docs
+
+## Development and testing
+
+### Interacting with the MinIO blob store
+
+To peek inside the MinIO store, use the MinIO client (`mc`):
+
+```
+kubectl run minioclient -i --tty --rm --image minio/mc -n kubeflow --command -- /bin/sh
+mc alias set myalias http://minio.kubeflow:9000 ACCESSKEY SECRETKEY
+mc ls myalias
+```
+
+You can then use other `mc` commands to interact with the contents of the store.
+
+### Interacting with the KFP Database
+
+When debugging KFP, it is sometime useful to interact directly with the database (to confirm that an experiment was written to the DB, etc).  To get a client on-cluster to do so, try:
+
+```
+kubectl run dbclient -i --tty --rm --image mariadb -n kubeflow -- /bin/bash
+# (from inside session)
+mysql -h KFPDB_SVC_IP -u mysql -D mlpipeline -p 
+# (enter password, `password`)
```

Comment:
Does this mean I have to literally enter "password" as password? Is this something the user can get from running some commands?